draft-ietf-scitt-architecture.md

```diff
@@ -619,15 +617,17 @@ Hence, the Registry may contain both Transparent Statements and governance entri
 
 ## Verifying Transparent Statements {#validation}
 
-For a given Artifact, Verifiers take as trusted inputs:
+For a given Transparent Statement, Verifiers take as trusted inputs:
 
 1. the distributed identifier of the Issuer (or its resolved key manifest),
-2. the expected name of the Artifact (i.e., the Feed),
+2. the collection of Transparent Statements to which this Statement about the Artifact belongs (i.e., the Feed),
 3. the list of service identities of trusted Transparency Services.
 
 When presented with a Transparent Statement for an Artifact, Consumers verify its Issuer identity, signature, and Receipt.
 They may additionally apply a validation policy based on the protected headers present both in the Envelope, the Receipt, or the Statement itself, which may include security-critical or Artifact-specific details.
 
+Some Verifiers may systematically fetch all Transparent Statements from the Feed and assess them alongside the Transparent Statement they are verifying to ensure freshness, completeness of evidence, and the promise of non-equivocation.
```

Comment:
Minor: This edit and all the others are great, but can we be a little more explicit about the what and/or how of the *promise* non-equivocation is in this context? The use of that word throws me off.

Reply:
The definition of non-equivocation is a good topic that we could possibly provide a link. For example: https://www.dpss.inesc-id.pt/~rodrigo/nonequiv_podc12.pdf

I think trying to describe non-equivocation in the spec is beyond the scope.

Reply:
Thanks for the paper, I reviewed it, it was informative (but I likely need more time to understand it). To bring it back to the PR, I was asking if we have to define this term in this spec. Other terms, such as client in #102.

If that is not needed, feel free to resolve this comment. I will take it to mean I might be behind the shared understanding of other WG contributors and participants and it is well understood (and I am asking us to define water for specs in a specification about water purification level of obtuseness, to be funny about it).

If you would like it be added, I can make an issue and PR an explanation summarized from this and/or other sources. Let me know.

Reply:
Don't agree as VEX style statements or claims can be augmented by the same entity and only the latter matters.  In that case the retrieval of all statements of a feed is a huge tax.  This concept of retrieval should be an application based solution (think a public registry) where graph style queries go.  It is at this layer the uber ownership and who gets to write new information per product is made.  That is not a function of the Transparency service.

Reply:
We presume the statement below, 

> the retrieval of all statements of a feed is a huge tax

with this reservation below.

> Don't agree as VEX style statements or claims can be augmented by the same entity and only the latter matters.

Given the higher-level architecture of SCITT, what about this sentence or current draft requires a consumer of a TS retrieve all statements of the feed at once or at all, but not look at one and query *if* there are statements that were published with metadata in one statement indicating others before or after this statement in a sequence? This goes outside the scope of SCITT, but inside VEX and other formats. In formats I have helped design or maintain, you can look at pointers to records, before, after, or show me the most current (from a data model perspective, not that different from how the IETF Datatracker works when you visit an old draft; you do not need to visit a page and read all drafts in a single page at once). So where is this called and why do we assume this here or elsewhere in the current spec draft?

Reply:
@aj-stein-nist 
> feel free to resolve this comment. I will take it to mean I might be behind the shared understanding of other WG contributors and participants and it is well understood (and I am asking us to define water for specs in a specification about water purification level of obtuseness, to be funny about it).
> If you would like it be added, I can make an issue and PR an explanation summarized from this and/or other sources. Let me know.

You're not behind, rather it's a great topic to clarify. Please do create an issue, and/or add a PR, it would be much appreciated as I'm sure others have the same question.

Reply:
@roywill 
> Don't agree as VEX style statements or claims can be augmented by the same entity and only the latter matters. In that case the retrieval of all statements of a feed is a huge tax. 

I agree with fetching all statements for an Artifact, based don't the `CWT_Claim => sub` may not be practical, or needed. 
The verifier may want all statements, of a particular contentType, such as VEX, or all statements of a specific issuer, of a specific contentType. 
These are all great topics when we discuss indexing, querying, etc. Which may likely be application layer concepts. 
We can iterate this text a bit more, in a subsequent issue/PR